silverstripe/silverstripe-versioned-admin
# ENH Do not use deprecated lifecycle methods

Issue https://github.com/silverstripe/silverstripe-admin/issues/1414

To test:
- Use the page history viewer
- Click on an item
- The `Content, Settings, History` tabs should have the correct alignment - they should not "overlap" the page preview
